bundles/org.openhab.binding.dsmr/src/main/resources/OH-INF/thing/meter_electricity_emucs_v1_0.xml

```diff
@@ -36,6 +36,8 @@
 			<channel id="emeter_instant_voltage_l1" typeId="instantVoltageL1Type"/>
 			<channel id="emeter_instant_voltage_l2" typeId="instantVoltageL2Type"/>
 			<channel id="emeter_instant_voltage_l3" typeId="instantVoltageL3Type"/>
+			<channel id="emeter_actual_demand" typeId="actualDeliveryType"/>
+			<channel id="emeter_maximum_demand_current_month" typeId="actualDeliveryType"/>
```

Comment:
You should create a specific channel type in the `channeltypes_electricity.xml`. The way it's done here means in the user interface the user will see 3 times the same channel label and not be able to distinguish from the label what the channel is for.

Also put the channels in logical position in the list of channels, and not just add them add the end of the list of channels. For example after the `emeter_actual_production` channel.

The `emeter_maximum_demand_current_month` also as a timestamp. You could also add this as a channel. (Same channel name with `_timestamp` appended: `emeter_maximum_demand_current_month_timestamp`)